data/bash-completion/fwupdmgr

```diff
@@ -65,7 +65,7 @@ _fwupdmgr_opts=(
 _show_filters()
 {
 	local flags
-	flags="$(command @libexecdir@/fwupdtool get-device-flags 2>/dev/null)"
+	flags="$(command fwupdtool get-device-flags 2>/dev/null)"
```

Comment:
I'm a bit confused; won't this run the system version not the one in the $PREFIX?

Reply:
the system version (the one in PATH) should be the one it uses, yes.  That's how it was before for fwupdagent too, no?

Reply:
For fwupdmgr I agree, as the system daemon will be running. For fwupdtool it's the local instance that makes more sense I think. I'm not super hung up on it as on practice it's going to return the right output 99% of the time.

Reply:
Here's the thing - if your PREFIX isn't having binaries in PATH, bash completion won't be installed somewhere that can be sourced anyway by default.

Reply:
That's very true; good point.